silverbackis/common-js-bundle/1.0/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+    "bundles": {
+        "JsSdkBundle\\JsSdkBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "GOOGLE_ANALYTICS_ID": "UA-XXXXXXXX",
```

Comment:
will all users need these 3 env variables?

Reply:
Not all users would, no, I'll comment them out to be included as wanted and adjust the post-install - thanks again.